inscriptions/ISic001402.xml

```diff
@@ -190,22 +195,17 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140910">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#AL">
                 <ab>
-                    <lb n="1"/>
-
-
-                    Σατορ
-                    <supplied reason="omitted">ν</supplied>
-                    <supplied reason="lost">ῖ</supplied>-<lb n="2"/>λε χρησ
-                    <supplied reason="lost">τὲ</supplied>
-                    <lb n="3"/>
-                    <supplied reason="lost">χα</supplied>ῖρ<unclear>ε</unclear>.
-                    <lb n="4"/>
+                    <lb n="1"/><persName type="attested"><name type="cognomen">Σατορ<choice><corr>ν</corr><sic>α</sic></choice><supplied reason="lost">ῖ</supplied>
```

Comment:
again, I'd go for the name in full normal here and note the reading of ΣΑΤΟΡΑ in the apparatus.